corehq/apps/app_manager/templates/app_manager/partials/settings/app_settings.html

```diff
@@ -184,17 +186,19 @@ <h6>Example:</h6>
             <div class="panel panel-appmanager panel-appmanager-form">
                 <form action="{% url "pull_master_app" domain app.id %}" method="POST">
                     {% csrf_token %}
-                    <legend>Update from
+                    <legend>
                         {% if upstream_url %}
-                            <a href="{{ upstream_url }}">master application</a>
+                            {% blocktrans %}Update from <a href="{{ upstream_url }}">master application</a>{% endblocktrans %}
                         {% else %}
-                            master application
+                            {% trans "Update from master application" %}
                         {% endif %}
                     </legend>
                     <div class="panel-body">
-                        <p> Your app is at version {{ app.version }}.
-                            {% if master_version %}The master app is at version {{ master_version }}.
-                            {% else %}Unable to get master version.
+                        <p>{% blocktrans with version=app.version %}Your app is at version {{ app.version }}.{% endblocktrans %}
```

Comment:
Confused by this line - if you needed to declare `version=app.version` shouldn't you be using `{{ version }}` inside the blocktrans?

Reply:
oh yeah, thanks, updated.